backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/transfer/HUTransformService.java

```diff
@@ -180,6 +182,7 @@ private HUTransformService(
 			@Nullable final List<TableRecordReference> referencedObjects)
 	{
 		this.huQRCodesService = SpringContextHolder.lazyBean(HUQRCodesService.class, huQRCodesService);
+		this.qrCodeConfigurationService = SpringContextHolder.lazyBean(QRCodeConfigurationService.class, null);
```

Comment:
what's the reason of lazyBean?
(asking because those are to be used in workarounds)

Reply:
pfuu, most probably we had that start up error with spring context not being initialized when it was trying to fetch that bean